src/components/App.js

```diff
@@ -1,104 +1,90 @@
-
-import React from 'react';
+import React from "react";
 import GradientItem from "./GradientItem";
-import '../App.css';
+import "../App.css";
 
 class App extends React.Component {
-    constructor() {
-        super();
-        this.state = {
-            inputValueFirst: '',//сюда должны получать значение из инпута
-            inputValueSecond: '',//сюда должны получать значение из инпута
-            isSubmitted: false,
-            isDisabled: false,
-            itemsList: []
-        };
-    }
-
-    handleChange = event => {
-        //console.log('input ', event.target.value);
-        this.setState({
-            [event.target.name]: event.target.value
-        });
-        // const { inputValueFirst, inputValueSecond } = this.state;
-
-        const regExpHexShort = new RegExp(/^#[0-9A-F]{3}$/gi);
-        const regExpHexLong = new RegExp(/^#[0-9A-F]{6}$/gi);
-        // console.log('input ', [event.target.name],event.target.value, );
-        if (!(regExpHexShort.test(event.target.value) || regExpHexLong.test(event.target.value))) {
-            // if(!(regExpHexShort.test(event.target.value) || regExpHexLong.test(event.target.value))) {
-            //
-            // }
-            console.log('not valid', event.target.value)
-        } else {
-            console.log(' valid', event.target.value);
-            // this.setState ({
-            //     isDisabled: !this.state.isDisabled
-            // })
-        }
+  constructor() {
+    super();
+    this.state = {
+      inputValueFirst: "", //сюда должны получать значение из инпута
+      inputValueSecond: "", //сюда должны получать значение из инпута
```

Comment:
Чтобы сделать нейминг понятным
```suggestion
      primaryColor: "", //сюда должны получать значение из инпута
      secondaryColor: "", //сюда должны получать значение из инпута
```